NEWS.md

```diff
@@ -1,5 +1,6 @@
-## 19.3.0 (in-progress)
+## 20.0.0 (in-progress)
 
+* No longer periodically excutes age to lost background processes (CIRC-1084)
```

Comment:
Typo, seems "executes" meant here.

Reply:
Thank you for noticing that. I've corrected it.